src/main/java/Duke.java

```diff
@@ -6,5 +10,117 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        Scanner sc = new Scanner(System.in);
+        ArrayList<Task> tasks = new ArrayList<>();
+
+        System.out.print("____________________\n"+
+                          "Hello! I'm Duke\n"+
+                          "What can I do for you?\n");
+        boolean isToQuit = false;
+        String input, startOfInput;
+
+        while(!isToQuit){
+            input = sc.nextLine();
+            String commandWord = getCommand(input);
+            String info = getInfo(input);
+
+            switch(commandWord){
```

Comment:
I like the use of a switch statement to split different commands into different functions! It is clear and simple to read.